tekton/utils_test.go

```diff
@@ -91,9 +91,9 @@ var _ = Describe("Utils", func() {
 				AsPipelineRun())).To(Equal(true))
 		})
 
-		It("returns true when ReleasePipelineRun.Status is `Succeeded` or false otherwise", func() {
+		It("returns true when PipelineRun.Status is `Succeeded` or false otherwise", func() {
 			releasePipelineRun.AsPipelineRun().Status.InitializeConditions(clock.RealClock{})
-			// MarkRunning sets Status to Unknown
+			// MarkReleasing sets Status to Unknown
```

Comment:
the comment change doesn't make sense to me. It is preceding `MarkRunning` not `MarkReleasing`

Reply:
Yep, mistake from the name refactoring. Fixed.